src/typescript/index.ts

```diff
@@ -14,6 +14,8 @@ async function main() {
     const files = await initiator.ListInputs()
     const mergeState: string = await initiator.GetValue("merge-state")
     const openapiType: string = await initiator.GetValue("openapi-type")
+    const subType: string = await initiator.GetValue("openapi-subtype")
```

Comment:
subType means?

Reply:
Currently used by rpass and passed through by autorest .  If existed,  we only need to run the rpass rules